opus/src/redux/reducer.js

```diff
@@ -2,7 +2,7 @@ const initialState = {
   token: null
 };
 
-const reducer = (state = initialState, action) => {
+const AuthReducer = (state = initialState, action) => {
```

Comment:
This is still a function, not a Component.

Plus, even though it's working for now, you might want to change names where this reducer is imported (index.js ?)